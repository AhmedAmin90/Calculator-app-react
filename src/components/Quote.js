import React from 'react';
import './About.css';

const Quote = () => (
  <div className="About">
    <div className="hero-section-text">
      <h1>Quote of the day</h1>
      <p className="my-description">
        Some people want it to happen, some wish it would happen, others make it happen.

      </p>
    </div>
  </div>
);

export default Quote;
