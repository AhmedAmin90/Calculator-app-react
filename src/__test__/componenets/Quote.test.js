/* eslint-disable no-unused-expressions */
import React from 'react';
import { create } from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quote from '../../components/Quote';

it('Should renders correctly', () => {
  const elem = create(<Quote />).toJSON();
  expect(elem).toMatchSnapshot();
});

test('displays heading', () => {
  render(<Quote />);
  const heading = screen.getByText('Quote of the day');
  expect(heading).toBeInTheDocument;
});
