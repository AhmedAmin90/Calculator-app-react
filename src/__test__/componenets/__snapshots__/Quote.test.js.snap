exports[`test Should renders correctly 1`] = `
<div
  className="About">
  <div
    className="hero-section-text">
    <h1>
      Quote of the day
    </h1>
    <p
      className="my-description">
      Some people want it to happen, some wish it would happen, others make it happen.
    </p>
  </div>
</div>
`;
